### ISSUE TITLE
Combine AxisToAxis plugins

### ISSUE THREAD

**Issue by spadino** [Snoothy/UCR#47]
As per our Discord conversation, I suggest a kind of logical grouping for Axes. I propose a modifications to actual AxisToAxis plugin to be able to accept 1 to 3 input/output - linear, circular, spherical movements - so to avoid cluttering the select dropdown with three different plugins.

**Comment by evilC** [comment]
I don't think it could be done like this without fairly major changes - how many inputs a plugin has is controlled by the source code of the plugin.  
This is also potentially less efficient at run-time, as more checks need to be made etc.  
Additionally, when moving from 1 to 2 to 3 axes, you may well want different DeadZone code etc (AxesToAxes has a circular dz).
